pkg/ccl/sqlproxyccl/acl/cidr_ranges.go

```diff
@@ -50,11 +50,6 @@ func (p *CIDRRanges) CheckConnection(ctx context.Context, conn ConnectionTags) e
 		}
 	}
 
-	// By default, connections are rejected if no ranges match the connection's
-	// IP.
-	return errors.Newf(
-		"connection to '%s' denied: cluster does not allow public connections from IP %s",
-		conn.TenantID.String(),
-		conn.IP,
-	)
+	// By default, connections are rejected if no ranges match the connection's IP.
+	return errors.Newf("cluster does not allow public connections from IP %s", conn.IP)
```

Comment:
why remove the tenant ID here and elsewhere?

Reply:
Here's an example of a test log line before this change:

```
I241113 19:58:00.608198 1821 ccl/sqlproxyccl/server.go:347  [client=a.b.c.d:59939,tenant=123,cluster=foo-bar] 604  connection closed: connection blocked by access control list: connection to '123' denied: cluster does not allow public connections from IP a.b.c.d
```

The log message currently repeats the tenant ID (i.e. one in logtags, and another within the message), so repeating the tenant ID in the error message is redundant. Similarly, while the client's IP address could be inferred from the logtags, I've left it in the message for consistency with the format used when blocking private connections:

```
return errors.Newf("cluster does not allow private connections from endpoint '%s'", conn.EndpointID)
```

No strong opinions here, but let me know if you have thoughts.